packages/gatsby-cli/src/issue/telemetry.js

```diff
@@ -0,0 +1,81 @@
+const path = require(`path`)
+const fs = require(`fs-extra`)
+
+const envInfo = require(`../env-info.js`)
+
+const mdCodeBlock = (content, lang) =>
+  `
+\`\`\`${lang}
+${content}
```

Comment:
I've solved this problem differently before, e.g.

```js
const mdCodeBlock = (content, lang) => [
  '```' + lang,
  content,
  '```'
].join('\n').trim()
```

Reply:
Since eslint wants all strings to be template strings anyways, I figured my solution would be cleaner than

```js
const mdCodeBlock = (content, lang) => [
  `\`\`\`` + lang,
  content,
  `\`\`\``
].join(`\n`).trim()
```

But of course I can also use yours, and maybe turn off the eslint rule for that function.

Reply:
I agree with you here. Keep what you had!